CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
+- "Reset filters" button for campaigns
+
+### Fixed
+- Campaign filters retains state when navigating away from the campaigns search
```

Comment:
```suggestion
- State retains page number and filters when navigating to and from individual campaign cards
```

Reply:
Yeah but this has to be specific to campaigns because there are filters on the users page that don't have this logic.

Reply:
@kamicut updated